openmetadata-docs/content/v1.1.0/deployment/docker/index.md

```diff
@@ -74,35 +77,88 @@ Follow the instructions [here](https://docs.docker.com/compose/cli-command/#inst
+- Switch Branch to release version which will be `1.1.0`
+- Browse to `docker` > `docker-compose-openmetadata` directory
+- Download the Docker Compose and env files available in the directory
+
+To do this via terminal, just run the below commands -
 
 ```commandline
-docker compose up -d 
+wget https://github.com/open-metadata/OpenMetadata/blob/1.1.0/docker/docker-compose-openmetadata/docker-compose-openmetadata.yml
+wget https://github.com/open-metadata/OpenMetadata/blob/1.1.0/docker/docker-compose-openmetadata/env-mysql
 ```
-This command will pull the docker images of Openmetadata for MySQL, OpenMetadat-Server, OpenMetadata-Ingestion and Elasticsearch.
 
-Upon running this command you should see output similar to the following.
+If you are looking for Docker compose file with PostgreSQL, the command to download relevant files will be
+
 ```commandline
-+] Running 7/8
- ⠿ Network metadata_app_net                        Created                                                                                               0.2s
- ⠿ Volume "metadata_ingestion-volume-dag-airflow"  Created                                                                                               0.0s
- ⠿ Volume "metadata_ingestion-volume-dags"         Created                                                                                               0.0s
- ⠿ Volume "metadata_ingestion-volume-tmp"          Created                                                                                               0.0s
- ⠿ Container openmetadata_elasticsearch            Started                                                                                               5.9s
- ⠿ Container openmetadata_mysql                    Started                                                                                              38.3s
- ⠿ Container openmetadata_server                   Started                                                                                             124.8s
- ⠿ Container openmetadata_ingestion                Started                                                                                               0.3s
+wget https://github.com/open-metadata/OpenMetadata/blob/1.1.0/docker/docker-compose-openmetadata/docker-compose-openmetadata-postgres.yml
+wget https://github.com/open-metadata/OpenMetadata/blob/1.1.0/docker/docker-compose-openmetadata/env-postgres
+```
+
+### 3. Update Environment Variables required for OpenMetadata
+
+In the previous [step](#2-download-docker-compose-file-from-github-release-branch), we download two files, one is `docker-compose` and another is `environment file` required for docker compose.
+
+Identify and update the environment variables in the file to prepare openmetadata configurations.
+
+For example, we want to configure external database and search engine configurations. In that case, you will update the below section of environment variable file -
+
+```bash
+...
+# Database configuration for MySQL
```

Comment:
we need to include configuration for MySQL and Postgres , Elastic and OpenSearch

Reply:
This is done and updated in the PR!